compiler/codegen/symbol.cpp

```diff
@@ -43,7 +43,6 @@
 
 // LLVM debugging support
 #include "llvmDebug.h"
-
```

Comment:
Let's put this space back, since it doesn't have to do with your intended change set.